tests/test_general.py

```diff
@@ -707,34 +706,38 @@ def test_error_advanced(self):
         z = Variable('z')
         model = {z: a * log(b * x + c * y)}
 
-        # fit = Fit(model, xy, z, absolute_sigma=False)
-        fit = Fit(model, xdata, ydata, zdata, absolute_sigma=False)
-        # fit = Fit(model, x=xdata, y=ydata, z=zdata, absolute_sigma=False)
+        # Use a gradient model because Mathematica uses the Hessian
+        # approximation instead of the exact Hessian.
```

Comment:
That's interesting, since Mathematica does have access to the Hessian... Using the Hessian is supposed to be more accurate, right?

Reply:
Yes. I think it might be a convention/computational ease thing?

Reply:
I've not caught Mathematica on either of those before. So I find it rather worrying.